docs/resources/apig_group.md

```diff
@@ -33,56 +33,79 @@ resource "huaweicloud_apig_group" "test" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) Specifies the region in which to create the API group resource. If omitted,
-  the provider-level region will be used. Changing this will create a new API group resource.
+* `region` - (Optional, String, ForceNew) Specifies the region where the dedicated instance is located.  
```

Comment:
s/dedicated instance/API group

Reply:
the apig group is belong to the dedicated instance